launch/map4_localization_launch1/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.14)
+project(map4_localization_launch1)
```

Comment:
> Can you briefly explain why you named map4_localization_launch1, and not something like map4_localization_launch?

I wanted to clarify that there are multiple map4 localization launches, but I don't have a strong argument.

> It would be nice if you can summarize why you added map4_localization_launch instead of sticking to tier4_localization_launch. (Maybe also adding an internal link to the slack discussion thread)

OK
I am going to summarize below.
https://github.com/autowarefoundation/autoware.universe/pull/3063#issue-1620972228

Reply:
Cool, thanks!

How about just creating `map4_localization_launch` and write all the if-conditions (for eagleye_pose_twist and eagleye_twist configuration) there?

Reply:
@kminoda 
Just to confirm, is my understanding correct that that is proposal X?

proposal X
 - map4_localization_launch package
   - eaglleye_twist_localization_launch directry
     - localization.launch.xml
   - eagleye_pose_twist_localiation_launch directry
     - localization.launch.xml
 
proposal Y
 - map4_localization_launch
   - localization.launch.xml (Conditional branching with parameter `pose_estimator_mode`.<-This is complicated.)

Reply:
My intention was Plan Y, but either is fine for me (since it's "map4_localization_launch")

Reply:
I fixed it.